### DISCUSSION TITLE
Getting 404 against your File Upload code

### DISCUSSION THREAD

**Discussion by danishmahmood83** [graphql-java-kickstart/samples#265] [Q&A]
![image](https://user-images.githubusercontent.com/65555674/236181090-40b03cad-1757-4af5-b67f-74bab96f6fd2.png)

**Comment by federicorispo** [comment]
Hi @danishmahmood83, thanks for the feedback! 

With the PR #321 we updated the file-upload example and we added a README explaining how to run it.

I hope the problem is solved!
